docs/sources/write/style-guide/write-for-developers/index.md

```diff
@@ -85,6 +85,14 @@ Additional tips:
   For example, don't change `MyEvent` to `MyEvents`; refer to the `MyEvent` objects.
 - If the element does some sort of action, start the first sentence of the description with an action verb.
 
+### Reference structure
+
+Organize references alphabetically, organizing by frequency of use is too subjective.
+
+If the reference has required and optional components, document them separately.
```

Comment:
Do we interpret this as... put the required at the top of the list, or as a separate section?

I'm not yet 100% on putting required as a separate section.

As a separate section it could look like this:
![image](https://github.com/user-attachments/assets/f4b0f338-899d-45a5-97c9-c003d38cb2fa)

Reply:
Yeah, I'm not certain either. Might be easiest to resolve this discussion in Slack and then update this PR with the outcome and reasons.

Reply:
Maybe add required or optional in the description, as the first bit of information. Like "Required. Handler from an..."? I think that's how I've done this in the past.